arangodb-net-standard/DatabaseApi/DatabaseApiClient.cs

```diff
@@ -34,13 +34,21 @@ public async Task<PostDatabaseResponse> PostDatabaseAsync(PostDatabaseBody reque
             }
         }
 
-        public async Task<DeleteDatabaseResponse> DeleteDatabaseAsync(string dbName)
+        /// <summary>
+        /// Delete a database. dropping a database is only possible from within the _system database.
+        /// The _system database itself cannot be dropped.
+        /// DELETE /_api/database/{database-name}
+        /// </summary>
+        /// <param name="dbName"></param>
+        /// <returns></returns>
+        public async Task<DeleteDatabaseResponse> DeleteDatabaseAsync(string databaseName)
         {
-            using (var response = await _client.DeleteAsync(_databaseApiPath + "/" + WebUtility.UrlEncode(dbName)))
+            using (var response = await _client.DeleteAsync(_databaseApiPath + "/" + WebUtility.UrlEncode(databaseName)))
             {
                 if (response.IsSuccessStatusCode)
                 {
-                    return new DeleteDatabaseResponse((int)response.StatusCode);
+                    var stream = await response.Content.ReadAsStreamAsync();
+                    return new DeleteDatabaseResponse((HttpStatusCode)response.StatusCode);
```

Comment:
Shouldn't we return the deserialized reponse?

Reply:
Frustratingly, I cannot get the serializer to serialise the result. It always returns 0, false, false.
But if you try just using an object to map it, the string result is correct but in camel case property names. So I tried putting the camel case resolver back into the DeserializeJsonFromStream method but that still did not work. So I'm at a loss on how to resolve